compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/node/RootNodeOwner.skiko.kt

```diff
@@ -23,6 +23,7 @@ import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
```

Comment:
Do we have numbers how much it increases the performance?

Reply:
It removes the degradation on `LazyGrid` benchmark from 9.8 ms per frame to 7.1 ms. `VisualEffects` is still affected (but Google confirms that it is a kind expected for such an artificial example)